changelogs/fragments/67089-sns_topic-notfound-backoff.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- sns_topic - Add backoff when we get Topic NotFound exceptions which listing the subscriptions.
```

```suggestion
- sns_topic - Add backoff when we get ``Topic NotFound`` exceptions which listing the subscriptions.
```
Also, `which` -> `when`?